easybuild/easyconfigs/c/CondaCreate/requirements.txt

```diff
@@ -0,0 +1 @@
+perl-app-cpanminus
```

Comment:
Thanks for the point in the right direction. ;-)

I think if people are creating conda environments they know being on the internet is required.

I'm not sure what you mean by that.

```
conda install my-package #defaults to latest
conda install my-package=1.2 #install a specific version
```

Actually, let me make sure the config and easyblocks I have here are up to date. I did some work on them recently and I'm not sure I pushed that work.